service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/registry/AbstractServiceRegistry.java

```diff
@@ -263,6 +263,7 @@ public void destroy() {
 
   @Override
   // TODO: this is for 3rd party invocation, and a better way can be provided
+  // TODO:  microserviceManager.getVersionsByName() can not be used, will delete this in SCB-1949
```

Comment:
3rd party is a good mechanism， must not delete it.

Reply:
OK. I'll fix the problem in next PR without delete the interface.